docker-compose.template.yml

```diff
@@ -0,0 +1,61 @@
+version: '3.2'
+services:
+  invoices:
+    image: invoices
+    user: "node"
+    working_dir: /usr/src/app
+    command: "node ./api/api-invoices/src/index.js"
+    restart: always
+    ports:
```

Comment:
you don't need to bind ports since you use traefik

Reply:
I don't use traefik on local

Reply:
Your docker-compose is for the production purpose here.
So if you want to test something on local create an other docker-compose.